examples/wheel/lib/BUILD.bazel

```diff
@@ -21,12 +21,31 @@ licenses(["notice"])  # Apache 2.0
 py_library(
     name = "simple_module",
     srcs = ["simple_module.py"],
```

Comment:
Can we start leaving verbose comments in the examples?

Reply:
Could help with those? I'm not the best person with the example comments.